src/components/widgets-layouts/custom/accounts-invoices-list/accounts-invoices-list.directive.coffee

```diff
@@ -68,8 +68,12 @@ module.controller('WidgetAccountsInvoicesListCtrl', ($scope, $q, ImpacRoutes, Bo
   $scope.trxList.changeOverdueFilter = (overdueFilter) ->
     return if overdueFilter == $scope.trxList.overdue
     $scope.trxList.overdue = overdueFilter
+    todays_date = new Date()
+    formated_date = [todays_date.getFullYear(), (todays_date.getMonth() + 1), todays_date.getDate()].join('-')
```

Comment:
What do you think to use momentjs here? We use it a lot and easy you could do
`todays_date = moment().format('YYYY-MM-DD')`.